content/en/building/guides/database/querying_training_card_telemetry.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Querying Training Card Telemetry Data"
+linkTitle: "Querying Training Card Data"
+weight:
+description: >
+  How to use SQL queries to view metrics about Training Card usage
+relatedContent: >
+  building/guides/performance/telemetry/
+  core/overview/data-flows-for-analytics
+  building/guides/training/training-cards
+aliases:
+   - /apps/guides/database/querying_training_card_telemetry
+---
+
+Introduced in `4.2.0`, CHT has supported deployment of in-app training cards to facilitate remote training.
+
+Since interaction with training cards logs [telemetry data]({{< ref "building/guides/performance/telemetry" >}}), it is possible to view the data directly from CouchDB. However, it is more useful when you can run queries that provide useful metrics about the usage of training cards aggregated across many users, and interactions. With this in mind, it is typically easier to query the data using SQL from an [analytics database]({{< ref "core/overview/data-flows-for-analytics" >}}).
```

Comment:
Maybe mention that is when having couchdb data replicated using CHT Sync to a posgresql instance so then they can do sql queries